docs/api.md

```diff
@@ -475,21 +477,21 @@ coll.create({title: "foo"}).then(_ => coll.sync())
 
 ### Local transformers
 
-In a near future, Kinto.js will provide transfomers aimed at providing facilities to encode and decode records when persisted locally; you'll have to extend from `LocalTransformer` though.
+In a near future, Kinto.js will provide transfomers aimed at providing facilities to encode and decode records when persisted locally.
```

Comment:
Maybe lump local transformers and [collection transformers](https://github.com/Kinto/kinto.js/issues/154) into one "transformers in general" remark, and move that to line 148 (inside "Transformers" section, but before "Remote transformers" section)